src/game.cpp

```diff
@@ -2920,6 +2920,7 @@ bool game::handle_action()
             break;
 
         case ACTION_FIRE:
+            // TODO: Move handling this to a new function
```

Comment:
If you change the `TODO` to `@todo`, it shows up in documentation, making it easier to track.